openforcefield/utils/toolkits.py

```diff
@@ -1347,12 +1347,14 @@ def to_openeye(molecule, aromaticity_model=DEFAULT_AROMATICITY_MODEL):
             oebond = oemol.NewBond(oemol_atoms[atom1_index],
                                    oemol_atoms[atom2_index])
             oebond.SetOrder(bond.bond_order)
-            oebond.SetAromatic(bond.is_aromatic)
+            #oebond.SetAromatic(bond.is_aromatic)
```

Comment:
Same as above

Reply:
Added a TODO item to revisit the relationship between OFFMols' and OEMols' aromatic flags in a96274f